source/help/messaging/formatting-text.rst

```diff
@@ -125,23 +126,32 @@ Renders as:
 Code Block
 ----------
 
-Create a code block by indenting each line by four spaces, or by placing ``````` on the line above and below your code.
+Creating a fixed width code block is recommended for pasting multi-line blocks of code or other text output that is easier to read with fixed width font alignment. Examples include block text snippets, ASCII tables, and log files.
 
-Example:
+This can be accomplished in one of two ways:
 
-.. code-block:: none
+ 1. Placing 3x backtics :kbd:`\`\`\`` on the line directly above and directly below your code
 
+.. code-block:: none
 
   ```
+  this is my
   code block
   ```
 
-Renders as:
+
+* :guilabel:`TIP` - Enter 3x backticks :kbd:`\`\`\``, :kbd:`SHIFT + ENTER`, :kbd:`<type_your_code>`, :kbd:`SHIFT + ENTER`, and finally 3x backticks :kbd:`\`\`\``.
```

Comment:
Im not sure if Sphinx can handle this, LI please watch for this when we merge

Reply:
Seems to have merged fine @esethna

Reply:
Note I built and tested with sphinx prior to upload, syntax worked great :)